packages/compass-crud/src/stores/crud-store.js

```diff
@@ -117,6 +118,17 @@ export const setIsReadonly = (store, isReadonly) => {
   store.onReadonlyChanged(isReadonly);
 };
 
+/**
+ * Set the isPreferencesReadonly flag in the store.
+ *
+ * @param {Store} store - The store.
+ */
+export const setIsPreferencesReadonly = (store) => {
```

Comment:
I think it might be good to change the name and description to reflect the fact that this adds a listener, and doesn’t set anything on its own.

Reply:
This is similar to `setIsReadonly` in the same file, I think it means here that it sets this value in the state of the store, not on the collection or settings themselves.

Reply:
But `setIsReadonly` actually sets a value in the state of the store, and this one doesn’t do so directly, it ‘only’ installs a listener that does?